File: ReptiCare/Weight+CoreDataProperties.m
//
//  Weight+CoreDataProperties.m
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

#import "Weight+CoreDataProperties.h"

@implementation Weight (CoreDataProperties)

+ (NSFetchRequest<Weight *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Weight"];
}

@dynamic date;
@dynamic uniqueID;
@dynamic weight;
@dynamic reptile;

@end

File: ReptiCare/Weight+CoreDataProperties.swift
//
//  Weight+CoreDataProperties.swift
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

import Foundation
import CoreData 

extension Weight {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Weight> {
        return NSFetchRequest<Weight>(entityName: "Weight");
    }

    @NSManaged public var date: NSDate?
    @NSManaged public var uniqueID: NSNumber?
    @NSManaged public var weight: NSNumber?
    @NSManaged public var reptile: Reptile?

}
